person.rb

```diff
@@ -0,0 +1,35 @@
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id, :rental
+  attr_accessor :name, :age
```

Comment:
- [ ] There is a typo in rental. it should be `:rentals` because that is your instance variable in initialize method. 
- [ ] Also Kindly consider adding the `:rentals`  in line `5` after `attr_accessor` because it is needed to be this way in order for the association to work.